index.html

```diff
@@ -84,197 +46,274 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+          </li>
+        </ol>
+        <p>
+          That’s it! Your OpenStack is ready.
+        </p>
       </div>
-      <div class="row">
-        <div class="col-8">
-          <h2>
-            Useful tips
-          </h2>
-          <h3>
-            Launch a VM
-          </h3>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="microstack.launch test" value="microstack.launch cirros --name test" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>
-            Your VM is up and running now. You should see a message like:
-          </p>
-          <p>Access your server with <code>ssh -i /home/ubuntu/.ssh/id_microstack cirros@10.20.20.3</code></p>
+      <div class="col-6 js-tab__content" id="tab-three__content" role="tabpanel" aria-labelledby="tab-three">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install Multipass for macOS</h3>
 
-          <h3>
-            Try a different version, or a beta, or a daily build
-          </h3>
-          <p>
-            Snaps are published in <a class="p-link--external" href="https://snapcraft.io/docs/channels">channels</a> which are made up of a track (or major version), and an expected level of stability. Try <code>snap info microstack</code> to see what versions are currently published. You can run
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="snap refresh --channel=latest/beta microstack" value="sudo snap refresh --channel=latest/beta microstack" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>
-            or
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="snap refresh --channel=1.11/stable microstack" value="sudo snap refresh --channel=1.11/stable microstack" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>
-            to get the required version.
-          </p>
-          <p>
-            For more information refer to the <a class="p-link--external" href="https://opendev.org/x/microstack/src/branch/master/README.md">MicroStack documentation</a> or <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/microstack-get-started#0">Get started with MicroStack tutorial</a>.
-          </p>
-        </div>
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/CanonicalLtd/multipass/releases" class="p-button--positive js-download"
+                data-os="mac">
+                <span class="p-link--external">Download Multipass for macOS</span>
```

Comment:
```suggestion
                Download Multipass for macOS
```
As above, when JS is working this doesn’t take you to a different site, so it’s not an external link.